resolwe/flow/management/commands/list_docker_images.py

```diff
@@ -18,21 +20,30 @@
 
 
 class Command(BaseCommand):
-    """List Docker images used by processes."""
+    """List Docker images used by processes.  Optionally also pull them."""
 
     help = "List Docker images used by processes in either plain text or YAML (for Ansible)"
 
     def add_arguments(self, parser):
         """Add an argument to specify output format."""
         parser.add_argument('--format', dest='format', default='plain',
                             help="Set output format ('plain' [default] or 'yaml')")
+        parser.add_argument('--pull', dest='pull', default=False, action='store_true',
+                            help="Pull all images with Docker")
+        parser.add_argument('--stop-if-pull-fails', dest='stop_if_pull_fails',
```

Comment:
In my opinion the default behaviour should be to stop on error. Then you can also use shorter name for this argument, e.g. `--ignore-errors`.

Reply:
I've implemented this now -- it fails by default, unless you give it the `--ignore-pull-errors` option or set the `FLOW_DOCKER_IGNORE_PULL_ERRORS` settings variable to `True` (or the `RESOLWE_DOCKER_IGNORE_PULL_ERRORS` environment variable to `1`).

By default, errors are ignored in the test environment, but not in production.  No changes are required to the `settings.py` file in production, since the code simply defaults to fail-on-error if those variables aren't set.